service_mesh/v2x/ossm-observability.adoc

```diff
@@ -33,3 +33,14 @@ endif::[]
 include::modules/ossm-access-grafana.adoc[leveloffset=+1]
 
 include::modules/ossm-access-prometheus.adoc[leveloffset=+1]
+
+include::modules/ossm-integrating-with-user-workload-monitoring.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_user-workload-monitoring"]
+== Additional resources 
+
+For information about monitoring for user-defined projects in {product-title}, see:
+ifndef::openshift-rosa,openshift-dedicated[]
+xref:../../monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
+endif::[]
```

Comment:
```suggestion
ifndef::openshift-rosa,openshift-dedicated[]
* xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc[Enabling monitoring for user-defined projects]
endif::[]
```

see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
Also, adoc not html 🙂